### ISSUE TITLE
Example for agents_to_aggregate error

### ISSUE THREAD

**Issue by skgallagher** [skgallagher/EpiCompare#6]
I'm having issues getting the new agent simulation SIR output to work with agent to aggregate.  Here are three reproducible examples without the agent simulation function.

```
## data.frame  
    test_df <- data.frame(agent_id = factor(1:5),
                          sim = 1,
                          tI = c(19, 18, NA, NA, NA),
                          tR = c(35, 27, NA, NA, NA))
    out <- agents_to_aggregate(test_df, c(tI, tR))
    expect_true("X0" %in% colnames(out))

## tbl_df version
    test_df <- data.frame(agent_id = factor(1:5),
                          sim = 1,
                          tI = c(19, 18, NA, NA, NA),
                          tR = c(35, 27, NA, NA, NA))
        test_tbl <- dplyr::tbl_df(test_df)
        out <- agents_to_aggregate(test_tbl, c(tI, tR))
        expect_true("X0" %in% colnames(out))


## grouped_df version
    test_df <- data.frame(agent_id = factor(1:5),
                          sim = rep(factor(c(1:2)), each = 5),
                          tI = c(19, 18, NA, NA, NA),
                          tR = c(35, 27, NA, NA, NA))
        test_tbl <- dplyr::tbl_df(test_df) %>% dplyr::group_by(sim)
        out <- agents_to_aggregate(test_tbl, c(tI, tR)) 
        expect_true("X0" %in% colnames(out))


```

@benjaminleroy

**Comment by benjaminleroy** [comment]
These should be corrected in fa2e197. @skgallagher please check that this is true.

**Comment by benjaminleroy** [comment]
This issue has been corrected in fa2e197.
